rotkehlchen/tests/api/test_misc.py

```diff
@@ -429,6 +429,7 @@ def test_events_mappings(rotkehlchen_api_server_with_exchanges):
     assert 'per_protocol_mappings' in result
     assert 'global_mappings' in result
     assert 'event_category_details' in result
+    assert 'accounting_events_icons' in result
```

Comment:
Since `AccountingEventType` is an Enum can you check that all the values are mapped? Otherwise I see that we will add a new type and forget adding the icon mapping

Reply:
Good idea